scss/base.scss

```diff
@@ -34,5 +34,15 @@ mark {
     overflow-wrap: break-word;
 }
 
+code {
```

Comment:
We might need to tweak this later if there can be (multiline) code blocks in the dataset descriptions, but looks good for now.

(If you want to restrict it to just the metadata specification page we could move this CSS or give it a specific class name.)